javascript/javascript/examples/AsyncAwait.js

```diff
@@ -3,9 +3,8 @@ async function f(){}
 class C {
     async method(){}
     static async method1(){}
-    async static #method2(){}
```

Comment:
Illegal class element according to [specification](https://tc39.es/ecma262/multipage/ecmascript-language-functions-and-classes.html#prod-ClassElement) - `static` keyword must be before method definition which may be async.